docs/getting-started.md

```diff
@@ -89,6 +89,15 @@ helm install flow-aggregator antrea/flow-aggregator --set clickHouse.enable=true
 helm install theia antrea/theia -n flow-visibility --create-namespace
 ```
 
+From Theia v0.3, [Theia Command-line Tool](theia-cli.md) uses Theia Manager
+as a layer to connect and manage other resources like ClickHouse and Spark
+Operator. To enable Theia Manager, please run the following command when
+installing Theia:
+
+```bash
+helm install theia antrea/theia --set sparkOperator.enable=true,theiaManager.enable=true -n flow-visibility --create-namespace
```

Comment:
Hi Yanjun, I take a look together with the above paragraphs and found we are missing `helm install flow-aggregator` here. 
Since I feel writing `helm install flow-aggregator antrea/flow-aggregator --set clickHouse.enable=true,recordContents.podLabels=true -n flow-aggregator --create-namespace` three times is a bit repetitive. Could we split the installation section of flow-aggregator and theia in these paragraphs? It may look better IMO.

Reply:
I also feel the commands repetitive though I remember the original purpose to put them together is to keep the installation in "one step". Anyway it still sounds good to me to split the step under the current condition. Updated, thanks!